pkg/resource/managed_reconciler_test.go

```diff
@@ -95,13 +88,9 @@ func TestManagedReconciler(t *testing.T) {
 				},
 				mg: ManagedKind(MockGVK(&MockManaged{})),
 				o: []ManagedReconcilerOption{
-					func(r *ManagedReconciler) {
-						r.external = mrExternal{
-							ExternalConnecter: ExternalConnectorFn(func(ctx context.Context, mg Managed) (ExternalClient, error) {
-								return &NopClient{}, errBoom
-							}),
-						}
-					},
+					WithExternalConnecter(ExternalConnectorFn(func(_ context.Context, mg Managed) (ExternalClient, error) {
```

Comment:
Haha, this makes me realise I spelled `Connector` two different ways. No need to fix it now though.